recaf-ui/src/main/java/me/coley/recaf/ui/jfxbuilder/component/control/field/TextFieldComponent.java

```diff
@@ -0,0 +1,29 @@
+package me.coley.recaf.ui.jfxbuilder.component.control.field;
+
+import javafx.beans.property.StringProperty;
+import javafx.beans.value.ObservableStringValue;
+import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.scene.control.TextField;
+import me.coley.recaf.ui.jfxbuilder.component.Component;
+import me.coley.recaf.ui.jfxbuilder.component.control.RegionComponent;
+
+public interface TextFieldComponent<Self extends TextFieldComponent<Self>>
+	extends RegionComponent<Self>, Component<TextField, Self> {
+
+	Self prompt(String prompt);
+
+	Self prompt(ObservableStringValue prompt);
+
+	Self text(String text);
+
+	Self bindText(StringProperty text);
```

Comment:
In all other cases it is just `bind`, can follow that pattern here if there are no conflicts.